src/components/shared/passphraseRenderer/passphraseRenderer.css

```diff
@@ -0,0 +1,131 @@
+@import '../../../app/variables.css';
+@import '../../../app/mixins.css';
+
+.header {
+  @mixin contentLargest bold;
+
+  color: var(--color-maastricht-blue);
+  display: flex;
+  margin: 0;
+}
+
+.subheader {
+  @mixin contentLargest;
+
+  text-align: left;
+  margin: 15px 0;
+  color: var(--color-slate-gray);
+}
+
+.passphraseContainer {
+  width: 100%;
+  background-color: var(--color-white);
+  border: 1px solid var(--dark-border-color);
+  flex-wrap: wrap;
+  padding: 12px 16px;
+  box-sizing: border-box;
+  counter-reset: indicator;
+
+  & .inputRow {
+    text-align: left;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  & .inputContainer {
+    margin: 6px 0;
+    display: flex;
+    justify-content: left;
+  }
+
+  & .inputValue {
+    @mixin contentLargest;
+
+    text-align: left;
+
+    &::before {
+      counter-increment: indicator;
+      content: counter(indicator) ". ";
+      color: var(--color-ghost);
+    }
+  }
+
+  & .default {
+    color: var(--color-maastricht-blue);
+  }
+
+  & .emptyInput {
+    color: var(--color-ghost);
+    cursor: pointer;
+    font-family: sans-serif;
+
+    &::before {
+      font-weight: var(--font-weight-bold);
+      color: var(--color-maastricht-blue);
+    }
+  }
+
+  & .emptyInputFocused {
+    color: var(--color-ultramarine-blue);
+    cursor: pointer;
+    font-family: sans-serif;
+
+    &::before {
+      font-weight: var(--font-weight-bold);
+      color: var(--color-ultramarine-blue);
+    }
+  }
```

Comment:
💅 Doesn't look very DRY. 
IMO would be more readable to combine common properties of `.emptyInput` `.emptyInputFocused`  `.emptyInput::before ` `.emptyInputFocused::before ` where possible.

Maybe separating `emptyInputFocused` into two clases `.emptyInput.focused` would make it easier not to repeat which properties are applied because the input is empty and which ones because it's focused.

Reply:
Also calling it `input` confused me at first. I wanted to recommend https://www.w3schools.com/cssref/sel_empty.asp and https://www.w3schools.com/cssref/sel_focus.asp instead of some classes.

Then I saw it's just a span with a word, so `emptyWord` would be IMO more descriptive class name.